redwood-treehouse-host/build.gradle

```diff
@@ -35,10 +35,21 @@ kotlin {
 
     commonTest {
       dependencies {
-        implementation libs.kotlin.test
         implementation libs.assertk
+        implementation libs.kotlin.test
         implementation libs.kotlinx.coroutines.test
+        implementation libs.okio
         implementation libs.turbine
+        implementation projects.redwoodLayoutTesting
+        implementation projects.redwoodLazylayoutTesting
+        implementation projects.testApp.schema.protocolHost
+        implementation projects.testApp.schema.testing
+      }
+      if (!rootProject.hasProperty('redwoodNoApps')) {
+        kotlin.srcDir('src/appsTest/kotlin')
+        dependencies {
+          implementation projects.testApp.presenterTreehouse
+        }
```

Comment:
I'm pretty sure this means this code isn't going to be tested on CI. You would need to add `:redwood-treehouse-host:check` to the 'test-app' shard.

Reply:
On my follow-up list.

Reply:
https://github.com/cashapp/redwood/pull/2065